src/stackedBars.ts

```diff
@@ -52,32 +58,44 @@ export function renderStackedBars(pareto: Pareto, settings: Settings) {
         .attr("shape-rendering", "crispEdges")
         .attr("stroke", (d) => (d.isMarked ? "#000" : "none"))
         .attr("stroke-width", (d) => (d.isMarked ? settings.style.selectionBox.strokeWidth : "0"));
-        
-    
-        
-     const range =  svgBoundingClientRect.width - resources.PADDINGRIGHT 
-   sel.append("line")
+
+    // Draw the 80/20 cutoff line
+    const range = svgBoundingClientRect.width - resources.PADDINGRIGHT;
+    sel.append("line")
         .style("stroke", "#FA7864")
         .style("stroke-width", resources.LINEWEIGHT)
         .style("stroke-dasharray", "8 8 ")
-        .attr("x1",  range)
+        .attr("x1", range)
         .attr("y1", percentageAxis(80))
-        .attr("x2",0)
+        .attr("x2", 0)
         .attr("y2", percentageAxis(80))
-        .on("click",selectedBars)
-        .on("mouseover",  showLineToolTip)
+        .on("click", selectedBars)
+        .on("mouseover", showLineToolTip)
         .on("mouseout", function () {
             settings.tooltip.hide();
-        })
-  
+        });
+
+    // Add invisible line to click on
+    sel.append("line")

```

Comment:
Can we have the invisible line prioritized on top and then remove clicking and hovering on the dashed line?

Reply:
I tried, but if we remove the clicking and hovering then we can't click on the dashed lines anymore only inbetween